samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties

```diff
@@ -70,6 +73,7 @@ in_ps=En la(s) bater\u00eda(s)
 last_mod=\u00daltima modificaci\u00f3n
 mv_p=Mover bater\u00eda
 mv_q=Mover pregunta
+sol_q=Solucci\u00f3n
```

Comment:
Same value as 'solution', and it says 'Solucción'. let's use one

Reply:
Solved

Reply:
I still see solucción, I'll change it if you don't mind!